modules/govuk/manifests/apps/content_store.pp

```diff
@@ -89,5 +89,8 @@
     "${title}-ERRBIT_API_KEY":
       varname => 'ERRBIT_API_KEY',
       value   => $errbit_api_key;
+    "${title}-PLEK_SERVICE_TARIFF_URI":
+      varname => 'PLEK_SERVICE_TARIFF_URI',
+      value   => $trade_tariff_uri;
```

Comment:
I don't believe this is enough to bring the variable in from hieradata. The variable should be namespaced too.